pkg/services/ngalert/api/tooling/definitions/prom.go

```diff
@@ -152,9 +152,10 @@ type AlertingRule struct {
 	// required: true
 	Name string `json:"name,omitempty"`
 	// required: true
-	Query         string  `json:"query,omitempty"`
-	Duration      float64 `json:"duration,omitempty"`
-	KeepFiringFor float64 `json:"keepFiringFor,omitempty"`
+	Query              string   `json:"query,omitempty"`
+	QueriedDatasources []string `json:"queriedDatasources,omitempty"`
```

Comment:
Nit: maybe a bit clearer what the string actually is?
```suggestion
	QueriedDatasourceUIDs []string `json:"queriedDatasourceUIDs,omitempty"`
```